### DISCUSSION TITLE
Welcome to zircon-ha-repository Discussions!

### DISCUSSION THREAD

**Discussion by chris-xinhai-li** [lichr/zircon-ha-repository#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Xieo** [comment]
Hi this is an intresting project and thank you for allowing a free personal usage .
Do you plan to add more controls,textures, wall/room controls, objects and object size controls to the Zircon-HA project?

**Reply by chris-xinhai-li** [reply]
Yes, absolutely — all the missing features you mentioned are already on our backlog, and we're working hard to bring them to life. Thanks so much for your interest in our project — stay tuned!
